components/hydrator/internal/subject/cert_subject.go

```diff
@@ -40,7 +40,8 @@ func NewProcessor(ctx context.Context, certSubjectMappingCache certsubjectmappin
 
 	for _, m := range mappings {
 		if err := m.Validate(); err != nil {
-			return nil, err
+			// in case the certificate subject mapping is invalid, only log an error and continue so the hydrator component can still start
+			log.C(ctx).Errorf("Certificate subject mapping with subject: '%s' is not valid: %s", m.Subject, err.Error())
```

Comment:
Since we no longer return an error in this func, do you think we can change it to return only `*processor` instead of `(*processor, error)`?